CONTRIBUTING.md

```diff
@@ -0,0 +1,77 @@
+General guidance
+================
+
+The usual principes of respecting existing conventions and making sure that your changes
+are in line with the overall product design apply when contributing code to Pyenv.
+
+**MacOS warning:** As of this writing, we cannot test things on the Apple Silicon platform because
+neither Github Actions supports it nor any of the maintainers have the necessary hardware.
+So be extra careful when adding logic for these environments since we won't be able to test it
+and will have to take your word for it.

```

Comment:
I actually do have an M1 pro macbook now. If you have apple silicon stuff you'd like me to verify, please tag me!

Reply:
Are you sure that it's good enough to not issue a warning?..

I can assign issues/request reviews to you only anything MacOS-related...
...but these are the vast majority of the issues being submitted at the moment!!

Do you really have the capacirty to deal with them?